src/components/Status.tsx

```diff
@@ -38,18 +38,21 @@ const Status = () => {
 
   return (
     <>
-      <p className='titleStatus'>
-        Sensor Status
-      </p>
-      <div className='table'>
-        <div>Name</div>
-        <div className='call'>Last Call</div>
-        <div className='data'>Last Data</div>
+      <div className="card">
+        <h2 className='cardTitle'>Sensor status</h2>
+        <div className='cardBody'>
+        <table>
+          <tr>
```

Comment:
Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.

Reply:
Thanks, good catch. I'll update now.